types/twine-sugarcube/extensions.d.ts

```diff
@@ -0,0 +1,468 @@
+         * $pies.pluck() // Removes and returns a random pie from the array
+         */
+        pluck(): T | undefined;
+
+        /**
+         * Randomly removes the given number of members from the base array and returns the removed members as a new array.
+         * @param want The number of members to pluck.
+         * @since SugarCube 2.20.0
+         * @example
+         * // Given: $pies = ["Blueberry", "Cherry", "Cream", "Pecan", "Pumpkin"]
+         * $pies.pluckMany(3) // Removes three random pies from the array and returns them as a new array
+         */
+        pluckMany(want?: number): T[];
+
+        /**
+         * Appends one or more unique members to the end of the base array and returns its new length.
+         * @param members The members to append.
+         * @since 2.21.0
+         * @example
+         * // Given: $fruits = ["Apples", "Oranges"]
+         * $fruits.pushUnique("Apples") // Returns 2; $fruits ["Apples", "Oranges"]
+         * $fruits.pushUnique("Plums", "Plums") // Returns 3; $fruits ["Apples", "Oranges", "Plums"]
+         */
+        pushUnique(...members: T[]): number;
+
+        /**
+         * Returns a random member from the array. Does not modify the original.
+         * @since SugarCube 2.0.0
+         * @example
+         * // Given: $pies = ["Blueberry", "Cherry", "Cream", "Pecan", "Pumpkin"]
+         * $pies.random() // Returns a random pie from the array
+         */
+        random(): T | undefined;
+
+        /**
+         * Randomly selects the given number of unique members from the array and returns the selected members as a new array.
+         * Does not modify the original.
+         * @param want The number of members to select.
+         * @since SugarCube 2.20.0
+         * @example
+         * // Given: $pies = ["Blueberry", "Cherry", "Cream", "Pecan", "Pumpkin"]
+         * $pies.randomMany(3) // Returns a new array containing three unique random pies from the array
+         */
+        randomMany(want?: number): T[];
+
+        /**
+         * Randomly shuffles the array.
+         * @since SugarCube 2.0.0
+         * @example
+         * // Given: $pies = ["Blueberry", "Cherry", "Cream", "Pecan", "Pumpkin"]
+         * $pies.shuffle() // Randomizes the order of the pies in the array
+         */
+        shuffle(): T[];
+
+        /**
+         * Prepends one or more unique members to the beginning of the base array and returns its new length.
+         * @param members The members to append.
+         * @since SugarCube 2.21.0
+         * @example
+         * // Given: $fruits = ["Oranges", "Plums"]
+         * $fruits.unshiftUnique("Oranges") // Returns 2; $fruits ["Oranges", "Plums"]
+         * $fruits.unshiftUnique("Apples", "Apples") // Returns 3; $fruits ["Apples", "Oranges", "Plums"]
+         */
+        unshiftUnique(...members: T[]): number;
+
+        // deprecated members
+        // tslint:disable:no-redundant-jsdoc-2
```

Comment:
Remove these from here./. There is fix for this coming/ already in master which @orta is looking into